pkg/parse/parse.go

```diff
@@ -154,9 +154,6 @@ func CommonBuildOptionsFromFlagSet(flags *pflag.FlagSet, findFlagFunc func(name
 		return nil, fmt.Errorf("invalid --shm-size: %w", err)
 	}
 	volumes, _ := flags.GetStringArray("volume")
-	if err := Volumes(volumes); err != nil {
-		return nil, err
-	}
```

Comment:
This function is called in Podman in cmd/podman/images/build. go.  Will the lack of this call mess something up there?  It might be good to do a vendor from your sandbox and spin up a PR on Podman to verify for sure.

However, if @nalind or @giuseppe have a differing opinion, go with their thinking.

Reply:
Created a PR to test this on CI https://github.com/containers/podman/pull/18579

Reply:
@TomSweeneyRedHat Podman PR passes.